docs/sources/send-data/alloy/examples/alloy-kafka-logs.md

```diff
@@ -0,0 +1,406 @@
+---
+title: Sending Logs to Loki via Kafka using Alloy
+menuTitle: Sending Logs to Loki via Kafka using Alloy
+description: Configuring Grafana Alloy to recive logs via Kafka and send them to Loki.
+weight: 250
+killercoda:
+  title: Sending Logs to Loki via Kafka using Alloy
+  description: Configuring Grafana Alloy to recive logs via Kafka and send them to Loki.
+  backend:
+    imageid: ubuntu
+---
+
+<!-- INTERACTIVE page intro.md START -->
+
+#  Sending Logs to Loki via Kafka using Alloy
+
+Alloy natively supports receiving logs via Kafka. In this example, we will configure Alloy to receive logs via Kafka using two different methods:
+- [loki.source.kafka](https://grafana.com/docs/alloy/latest/reference/components/loki.source.kafka): reads messages from Kafka using a consumer group and forwards them to other `loki.*` components.
+- [otelcol.receiver.kafka](https://grafana.com/docs/alloy/latest/reference/components/otelcol.receiver.kafka/): accepts telemetry data from a Kafka broker and forwards it to other `otelcol.*` components.
+
+<!-- INTERACTIVE ignore START -->
+
+## Dependencies
+
+Before you begin, ensure you have the following to run the demo:
+
+- Docker
+- Docker Compose
+
+{{< admonition type="tip" >}}
+Alternatively, you can try out this example in our interactive learning environment: [Sending Logs to Loki via Kafka using Alloy](https://killercoda.com/grafana-labs/course/loki/alloy-kafka-logs).
+
+It's a fully configured environment with all the dependencies already installed.
+
+![Interactive](/media/docs/loki/loki-ile.svg)
+
+Provide feedback, report bugs, and raise issues in the [Grafana Killercoda repository](https://github.com/grafana/killercoda).
+{{< /admonition >}}
+<!-- INTERACTIVE ignore END -->
+
+
+## Scenario
+In this scenario, we have a microservices application called the Carnivorous Greenhouse. This application consists of the following services:
+- **User Service:** Manages user data and authentication for the application. Such as creating users and logging in.
+- **Simulation Service:** Generates sensor data for each plant.
+- **Websocket Service:** Manages the websocket connections for the application.
+- **Bug Service:** A service that when enabled, randomly causes services to fail and generate additional logs.
+- **Main App:** The main application that ties all the services together.
+- **Database:** A database that stores user and plant data.
```

Comment:
Did we lose the bullet point for the Plant service?

Reply:
good catch I must have removed it accepting changes